src/gatsby/gatsby-node/pages/createMdxPages.ts

```diff
@@ -225,13 +225,25 @@ const queries = [
         recursive: true,
       });
 
+      // eslint-disable-next-line max-len
+      const selectExcerpt = (description: string, excerpt: string, limit = 200) => {
+        if (description) {
+          if (description.length <= limit) {
+            return description.replace(/\n/g,' ');
+          }
+          return description.slice(0, limit) + '...';
```

Comment:
This will probably need to cleaned up as well.  Also, `\n` was counted in the limit check, so you'll need to do the replacing before it, eg 
```
if (description) {
  const cleanDescription = description.replace(/\n/g, ' ');
  if (cleanDescription.length <= limit) {
    return cleanDescription;
  }
  // note that this is the ellipsis character, not 3 periods
  return cleanDescription.slice(0, limit) + '…';
}
```

Reply:
Thanks! Would I also need the following?
```
const cleanExcerpt = excerpt.replace(/\n/g,' ');
return cleanExcerpt;
```